README.md

```diff
@@ -72,42 +73,100 @@ To get redirection to these pages, or redirection to third-party services, consi
 
 Remember to edit and customize all the values like `"CHANGE_ME!"` in the configuration files, in `proxy_conf.yaml` and in plugins configurations.
 
+## Docker image
+
+![Docker image design](gallery/docker-design.svg)
+
+the official Satosa-Saml2SPID docker immage is available at [scolagreco/satosa-saml2spid](https://hub.docker.com/r/scolagreco/satosa-saml2spid)
```

Comment:
this will be published on a github italia package if you agree, in CD with github actions